rts/Lua/LuaShaders.cpp

```diff
@@ -963,11 +961,12 @@ int LuaShaders::GetActiveUniforms(lua_State* L)
 }
 
 
-/***
- * Returns the locationID of a shaders uniform. Needed for changing uniform
- * values with function `gl.Uniform`.
+/*** Get location for a shaders uniform.
+ *
+ * Needed for changing uniform values with function `gl.Uniform`.
```

Comment:
Maybe?

> Needed to get the location of the uniform, so that the uniform value could be changed using one of the function from the family of `gl.Uniform*` functions.